lib/pages/community/community_overview.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:lemmy_api_client/v3.dart';
+
+import '../../util/extensions/api.dart';
+import '../../util/goto.dart';
+import '../../util/intl.dart';
+import '../../util/observer_consumers.dart';
+import '../../widgets/avatar.dart';
+import '../../widgets/cached_network_image.dart';
+import '../../widgets/fullscreenable_image.dart';
+import 'community_follow_button.dart';
+import 'community_store.dart';
+
+class CommunityOverview extends StatelessWidget {
+  final FullCommunityView fullCommunityView;
+  const CommunityOverview(this.fullCommunityView);
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final shadow = BoxShadow(color: theme.canvasColor, blurRadius: 5);
+
+    return ObserverBuilder<CommunityStore>(
```

Comment:
Nothing is being observed (pretty sure it even shows you a warning in the console?)